File: server/router/yunguan/product/productCategory.go
package product

import (
	"github.com/gin-gonic/gin"
	v1 "ygpt/server/api/v1"
)

type ProductCategoryRouter struct {
}

func (s *ProductCategoryRouter) InitProductCategoryRouter(Router *gin.RouterGroup) {
	dataRouter := Router.Group("product/Category")
	supplyInfoApi := v1.ApiGroupApp.YunGuanApiGroup.ProductCategoryApi

	{
		dataRouter.POST("addCategoryInfo", supplyInfoApi.AddCategoryInfo)         // 添加数据
		dataRouter.DELETE("deleteCategoryInfo", supplyInfoApi.DeleteCategoryInfo) // 删除数据
		dataRouter.POST("queryCategoryInfo", supplyInfoApi.QueryCategoryInfo)     // 获取数据列表
	}
}

File: web/src/api/yunguan/product/productCategory.js
import request from "@/utils/request";

// 添加系统运行工具
export function addCategoryInfo(modelData) {
    return request({
        url: "product/Category/addCategoryInfo",
        method: "post",
        data: modelData,
    });
}


// 获取系统运行工具列表
export function getCategoryInfoList() {
    return request({
        url: "product/Category/queryCategoryInfo",
        method: "post",
    });
}

// 删除系统运行工具
export function deleteCategoryInfo(modelData) {
    return request({
        url: "product/Category/deleteCategoryInfo",
        method: "delete",
        data: modelData,
    });
}
